src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/custom.py

```diff
@@ -503,6 +503,18 @@ def _create_kubernetes(resource_group_name, deployment_name, dns_name_prefix, na
 
 def k8s_get_credentials(name=None, resource_group_name=None, dns_prefix=None, location=None, user=None,
                         path=os.path.join(os.path.expanduser('~'), '.kube', 'config'), acs_info=None):
+    """Create a new Acs.
```

Comment:
Like mentioned in the issue, should we mark `resource-group-name` as mandatory? also remove the `dns-prefix` and `location`?